src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -132,7 +132,12 @@ private static (JObject obj, string path, string name)[] GetDependencies(OpsConf
                 foreach (var ecma2Yaml in ecma2YamlConfig)
                 {
                     var ecma2YamlJObject = JsonUtility.ToJObject(ecma2Yaml);
-                    ecma2YamlJObject[nameof(ECMA2YamlRepoConfig.SourceXmlFolder)] = Path.GetRelativePath(buildSourceFolder, ecma2Yaml.SourceXmlFolder);
+                    var sourceXmlFolder = Path.GetRelativePath(buildSourceFolder, ecma2Yaml.SourceXmlFolder);
+                    if (!Directory.Exists(Path.GetFullPath(Path.Combine(docsetPath, sourceXmlFolder))))
```

Comment:
Consider check existence before calling EcmaToYaml? I would like the config adapter to only to config JSON conversion so if a user writes it in `docfx.yml` this directory check is still there?

Reply:
Then we need to check the existence in multiple places, such as `SourceMap`

Reply:
yep, `FileResolver.TryReadString` seems like a reasonable addition to have.